src/ames_housing/assets/ames_housing_features.py

```diff
@@ -3,11 +3,23 @@
 import pandas as pd
 from dagster import asset
 
-from ames_housing.constants import SELECTED_FEATURES, TARGET
-from ames_housing.utils import get_key_prefix
-
-
-@asset(io_manager_key="csv_io_manager", key_prefix=get_key_prefix())
+from ames_housing.constants import (
+    LAKEFS_BRANCH,
+    LAKEFS_DATA_PATH,
+    LAKEFS_REPOSITORY,
+    SELECTED_FEATURES,
+    TARGET,
+)
+
+
+@asset(
+    io_manager_key="csv_io_manager",
+    metadata={
+        "repository": LAKEFS_REPOSITORY,
+        "branch": LAKEFS_BRANCH,
+        "path": LAKEFS_DATA_PATH,
```

Comment:
Passing the options this way makes them non-parametrizable - or is that a deliberate choice? I could see scenarios where operating on a different repo or branch would be desirable.

Any reason not to make these parameters configurable on the IO manager instead? That would also reduce the potential for inconsistencies and save a bunch of copy-pasted code in the asset definitions.